java_tests/src/test/java/org/lsst/sal/SAL_AddedGenerics.java

```diff
@@ -3,36 +3,35 @@
 import org.junit.Assert;
 import org.junit.Test;
 
-/**
- * A few tests require CSC that doesn't use generics, so we use Script.
- */
-public class SAL_ScriptTest {
+public class SAL_AddedGenericsTest {
 
     /**
-     * Test a SAL component that does not have csc in AddedGenerics.
+     * Test topics for a SAL component that does not have "csc" in AddedGenerics.
+     *
+     * Use Script: one of the few SAL component that is not a CSC.
      */
     @Test
-    public void testNoCscGenerics() {
+    public void testNoCsc() {
```

Comment:
This method was called the same as in the python unit test so why rename it? Please make sure that this method is called the same as the corresponding python unit test.

Reply:
I updated the Python to modernize these tests for the AddedGenerics tag, and am trying to update the Java to match.

Reply:
My bad, all tests do have `Test` in their name. So please rename the file to match the class name.